tests/quality_pragmaitc/test_pragmatic.py

```diff
@@ -1,8 +1,38 @@
 from goald.quality.pragmatic.quality_evaluator import PragmaticQualityEvaluator
+from goald.quality.pragmatic.model.context import Context
+from goald.quality.pragmatic.model.quality_constraint import QualityConstraint
+from goald.quality.pragmatic.model.common_metrics import CommonMetrics
+from goald.quality.pragmatic.model.pragmatic import Pragmatic
+from goald.quality.pragmatic.model.comparison import Comparison
 
 pqe = PragmaticQualityEvaluator()
 
-def test_evaluate_empty():
-    quality = pqe.evaluateQuality("", {})
-    assert quality == None
+# def test_evaluate_empty():
+#    quality = pqe.evaluateQuality("", {})
+#    assert quality is None
 
+
+def test_shouldGetDifferentQualityConstraintsForDifferentContexts():
+    aContext = Context("c1")
+    anotherContext = Context("c2")
+
+    aQC = QualityConstraint(aContext, CommonMetrics.METERS,
+                            30, Comparison.LESS_OR_EQUAL_TO)
+    anotherQC = QualityConstraint(
+        anotherContext, CommonMetrics.METERS, 60, Comparison.LESS_OR_EQUAL_TO)
+
+    goal = Pragmatic(False)
+
+    goal.getInterpretation().addQualityConstraint(aQC)
+    goal.getInterpretation().addQualityConstraint(anotherQC)
+
+    fullContext = []
+    fullContext.append(aContext)
+
+    assert aQC in goal.getInterpretation().getQualityConstraints(fullContext)
+
+    anotherFullContext = []
+    anotherFullContext.append(anotherContext)
+
+    assert anotherQC in goal.getInterpretation(
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/gh/lesunb/p2psimpy/pullRequest?prid=4861978)